### PR TITLE
Identifiers in localized models

### PR DESCRIPTION
**Pull request by linawolf**
releases: main, 11.5

### CODE REVIEW

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst [hunk 1]
```diff
@@ -192,3 +192,30 @@ Besides these two, there are a few more annotations available, which will be use
 in other contexts. For the complete list of all Extbase
 supported annotations, see the chapter :ref:`extbase-annotations`.
 
+..  _extbase-model-localizedUid:
+
+Identifiers in localized models
+================================
+
+Domain models have a main identifier :php:`uid` and two additional properties
+:php:`_localizedUid` (for langauge overlays) and :php:`_versionedUid`
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst:201]:
```suggestion
:php:`_localizedUid` (for language overlays) and :php:`_versionedUid`
```

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst [hunk 2]
```diff
@@ -192,3 +192,30 @@ Besides these two, there are a few more annotations available, which will be use
...
+Depending on whether the
+:typoscript:`languageOverlayMode` mode is enabled (:typoscript:`true` or
+:typoscript:`'hideNonTranslated'`) or disabled (:typoscript:`false`),
+the identifier contains different values.
+
+When :typoscript:`languageOverlayMode` is enabled, then the :php:`uid`
+property contains the :php:`uid` value of the default language record,
+the :php:`uid` of the translated record is kept in the :php:`_localizedUid`.
+
++----------------------------------------------------------+-------------------------+---------------------------+
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst:213]: Can we also add an example for _versionedUid?

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst:213]: _versionedUid doesn't have anything to do with localization it is only used for workspaces

Comment (brotkrueml) [reply Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst:213]: Yup, but it is mentioned. And as there is no other information about the _versionedUid we can also explain it in this secton (and rename it to "Identifiers in models" or "Identifiers in localized/versioned models"

Comment (linawolf) [reply Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Model.rst:212]: I believe, language overlays are already very complicated as-is. Workspaces add a whole new level of complexity. Especially when you use both at once.  Also workspaces are not widely used

So I moved the Workspace info in a tip box and added a todo. If you are experienced with workspaces feel free to add documentation, I have no experience in that area..

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**